### DISCUSSION TITLE
Promote Bracket

### DISCUSSION THREAD

**Discussion by evroon** [evroon/bracket#697] [General]
Not a code change request, but more of a general todo.

I want to promote Bracket more, I think it has more potential since tournament systems are used a lot and this is basically the only free/open-source tournament system AFAIK.

I promoted in a few ways now:
- HackerNews: https://news.ycombinator.com/item?id=39417973
- Discord channel of Mantine in `showcase` channel
- Added an entry in awesome-selfhosted

More ideas to promote would be appreciated.
Promoting it yourself even more :)

**Comment by sevi418** [comment]
Is there a public project roadmap available? I am interested in contributing and would like to know more.

**Comment by evroon** [comment]
Good question, now I have a private GitHub project that contains the issues (not much more), but I don't update it that often. It only shows what I'm working on now, not much more in the future. Do you have a suggestion for how to create and maintain a roadmap?

**Comment by sevi418** [comment]
I think the Milestones part of GitHub issues could be useful, we could set clear milestones for this project, and use proper issue labels to keep track of it.
